lib/adapter.js

```diff
@@ -165,8 +165,20 @@ module.exports = (function () {
     // TODO: Implement `Model.drop` functionality.
     drop: function(connectionName, collectionName, relations, cb) { cb() },
     // TODO: Implement `Model.drop` functionality.
-    destroy: function(connectionName, collectionName, options, cb) { cb() },
-
+    // The destroy method accepts ID and deletes a single record. 
+    destroy: function(connectionName, collectionName, options, cb) { 
+      log.verbose('deleting ' + collectionName)
+      spawnConnection(connectionName)
+        .then(function (connection) {
+          connection.sobject(collectionName).destroy(options.where.Id, function(err, ret) {
+            if (err || !ret.success){  return console.error(err, ret); } 
```

Comment:
This will hang if an error occurs because we never hit the callback. Instead, omit this and pass any errors back through to the caller. 
```
connection.sobject(collectionName).destroy(options.where.Id, function(err, ret) {
  cb(err, _.omit(ret, 'success', 'errors'));
}
```